### ISSUE TITLE
LDAP connect failed: invalid server address

### ISSUE THREAD

**Issue by firasalkafri** [etianen/django-python3-ldap#71]
```
firas@MAChINE1-2-1:~/platform$ ./manage.py ldap_sync_users
LDAP connect failed: invalid server address
INFO:django_python3_ldap.ldap:LDAP connect failed: invalid server address
Traceback (most recent call last):
  File "./manage.py", line 116, in <module>
    execute_from_command_line([sys.argv[0]] + django_args)
  File "/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/local/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/local/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/local/lib/python2.7/site-packages/django/utils/decorators.py", line 145, in inner
    return func(*args, **kwargs)
  File "/local/lib/python2.7/site-packages/django_python3_ldap/management/commands/ldap_sync_users.py", line 22, in handle
    user=user,
  File "/usr/lib/python2.7/contextlib.py", line 36, in __exit__
    raise RuntimeError("generator didn't stop after throw()")
RuntimeError: generator didn't stop after throw()
```

**Comment by etianen** [comment]
The RuntimeError is odd. I'll look into it.

Meanwhile, it looks like your LDAP_AUTH_URL is incorrect, which is the root cause of this.

**Comment by etianen** [comment]
I think I've fixed the RuntimeError in 0.9.14, but I think your LDAP_AUTH_URL is still wrong.
